components/admin_console/admin_definition.jsx

```diff
@@ -957,6 +957,31 @@ const AdminDefinition = {
                         onConfigSave: (displayVal) => displayVal * MEBIBYTE,
                         isDisabled: it.not(it.userHasWritePermissionOnResource('environment')),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_BOOL,
+                        key: 'FileSettings.ExtractContent',
+                        label: t('admin.image.extractContentTitle'),
+                        label_default: 'Enable document search by content:',
+                        help_text: t('admin.image.extractContentDescription'),
+                        help_text_default: 'When enabled, supported document types are searchable by their content. Search results for existing documents may be incomplete until a data migration is executed.',
```

Comment:
Please link the words: "until a data migration is executed" to https://www.mattermost.com/file-content-extraction

Reply:
@jespino ^

Reply:
Done :)